examples/text-generation/utils.py

```diff
@@ -170,6 +175,10 @@ def setup_model(args, model_dtype, model_kwargs, logger):
             model = wrap_in_hpu_graph(model, hash_with_views=not args.skip_hash_with_views)
         else:
             model = wrap_in_hpu_graph(model)
+
+    if args.torch_compile:
+        model = get_torch_compiled_model(model)
```

Comment:
@kausikmaiti  Can we add model specific check as generation using torch.compile isn't verified on models

Reply:
Added model specific check in separate commit. Please review.